src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Wasm/WasmBase.PlatformNotSupported.cs

```diff
@@ -0,0 +1,97 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Runtime.CompilerServices;
+using System.Runtime.Intrinsics;
+
+namespace System.Runtime.Intrinsics.Wasm
+{
+    internal abstract class WasmBase
```

Comment:
Not critical for this PR, but just noting that the approved API surface differs slightly from what the proposal initially had: https://github.com/dotnet/runtime/issues/53730#issuecomment-1209770214

`WasmBase -> PackedSimd`, `Constant` is cut as you should just use `Vector128.Create(...)` instead, most other things just had the parameter names updated from `x`, `a`, `b`, and `imm` to be things like `left`, `right`, `value`, `index`, etc.

Reply:
> @radekdoulik btw, you can use `/azp run runtime-wasm-perf` to run all the wasm benchmarks from `dotnet/performance` with this PR.

nice to have this pipeline. it would not measure these changes though, as they are hidden behind `WasmSIMD` property, which is disabled by default.